portal-api/api-tests/v1/utilisation-report-service/fee-record-corrections/get-fee-record-correction.api-test.js

```diff
@@ -0,0 +1,109 @@
+const { HttpStatusCode } = require('axios');
+const {
+  UtilisationReportEntityMockBuilder,
+  FeeRecordCorrectionEntityMockBuilder,
+  FeeRecordEntityMockBuilder,
+  RECORD_CORRECTION_REASON,
+  CURRENCY,
+} = require('@ukef/dtfs2-common');
+const { SqlDbHelper } = require('../../../sql-db-helper.ts');
+const app = require('../../../../src/createApp');
+const { as, get } = require('../../../api')(app);
+const testUserCache = require('../../../api-test-users');
+const { withClientAuthenticationTests } = require('../../../common-tests/client-authentication-tests');
+const { withRoleAuthorisationTests } = require('../../../common-tests/role-authorisation-tests');
+const { PAYMENT_REPORT_OFFICER } = require('../../../../src/v1/roles/roles');
+
+console.error = jest.fn();
+
+describe('GET /v1/banks/:bankId/fee-record-correction/:correctionId', () => {
+  const getUrl = ({ bankId, correctionId }) => `/v1/banks/${bankId}/fee-record-correction/${correctionId}`;
+
+  let testUsers;
+  let barclaysBank;
+  let hsbcBank;
+  let aBarclaysPaymentReportOfficer;
+  let aHsbcPaymentReportOfficer;
+
+  const correctionId = 7;
+  const facilityId = '12345678';
+  const exporter = 'An exporter';
+  const reportedFees = {
+    currency: CURRENCY.GBP,
+    amount: 77,
+  };
+  const reasons = [RECORD_CORRECTION_REASON.OTHER];
+  const additionalInfo = 'Some additional info';
+
+  beforeAll(async () => {
+    await SqlDbHelper.initialize();
+    await SqlDbHelper.deleteAllEntries('UtilisationReport');
+
+    testUsers = await testUserCache.initialise(app);
+    aBarclaysPaymentReportOfficer = testUsers().withRole(PAYMENT_REPORT_OFFICER).withBankName('Barclays Bank').one();
+    aHsbcPaymentReportOfficer = testUsers().withRole(PAYMENT_REPORT_OFFICER).withBankName('HSBC').one();
+    barclaysBank = aBarclaysPaymentReportOfficer.bank;
+    hsbcBank = aHsbcPaymentReportOfficer.bank;
+
+    const aFeeRecord = new FeeRecordEntityMockBuilder()
+      .withFacilityId(facilityId)
+      .withExporter(exporter)
+      .withFeesPaidToUkefForThePeriodCurrency(reportedFees.currency)
+      .withFeesPaidToUkefForThePeriod(reportedFees.amount)
+      .build();
+    const aUtilisationReport = new UtilisationReportEntityMockBuilder().withBankId(barclaysBank.id).withFeeRecords([aFeeRecord]).build();
```

Comment:
By not using 'forReport' in the Fee record mock builder it will actually inside the builder create it's own report. If we are attaching it to a specific report we should therefore use the forReport constructor to avoid any unintentional connections with accidental additional reports etc.